### PR TITLE
New Application Workflow Update

### PR DESCRIPTION
**Pull request by davoodharun**
#### What's this PR do?
- implements upload feedback (with delete confirmation modal)
- Implements multiUpload feature for attachment-directive
- OMB text edits related to new login workflow
- Fix to email verification feedback success error
#### What are the relevant tickets?
#197, #469, #324, #495

### CODE REVIEW

**Comment by davoodharun** [comment]
@mgwalker sorry this PR was so large, I just had a lot of work that was related to each other, specifically the multi-upload and file upload feedback features. Let me know if there is anything I can do to help with the review.

**Comment by mgwalker** [comment]
Does this also include the work from #559?  That might explain why so much of this code looks familiar.  😆

**Comment by davoodharun** [comment]
@mgwalker yes sorry I forgot to mention that I closed that PR. Turns out we have to fix a few things related to multi uploading on the back end, so a few more commits coming soon.

File: DOL.WHD.Section14c.Web/src/modules/pages/employerRegistrationTemplate.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     </p>
   </div>
   <p>Your account is now the point of contact for the employer you just registered. </p>
-  <p>If you have any questions contact us at 14c-help@dol.gov.</p>
+  <p>If you have any questions contact us at <a>14c-help@dol.gov.</a></p>
```
Comment (mgwalker) [inline DOL.WHD.Section14c.Web/src/modules/pages/employerRegistrationTemplate.html]: Should this include a `href` with a `mailto`?

File: DOL.WHD.Section14c.Web/src/modules/pages/employerRegistrationTemplate.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
   </div>
   <p><strong>This employer already has a registered account.</strong></p>
   <p>Please contact {{previouslyRegistered.name}}, the point of contact for this employer, to request access to the account.</p>
-  <p>If you have any questions, you can reach us at 14c-help@dol.gov.</p>
+  <p>If you have any questions, you can reach us at <a>14c-help@dol.gov</a>.</p>
```
Comment (mgwalker) [inline DOL.WHD.Section14c.Web/src/modules/pages/employerRegistrationTemplate.html]: Another `href="mailto..."`?

File: DOL.WHD.Section14c.Domain/Models/Submission/EmployerInfo.cs [hunk 3]
```diff
@@ -58,11 +59,24 @@ public EmployerInfo()
...
+                        x =>
+                            new EmployerInfoSCAAttachment
+                            {
+                                SCAAttachmentId = x,
+                                EmployerInfoId = Id,
+                            }).ToList();
+                }
+            }
+        }
+        public virtual ICollection<EmployerInfoSCAAttachment> SCAAttachment { get; set; }
```
[the remark this answers is not in the document]
Comment (dciceman) [reply DOL.WHD.Section14c.Domain/Models/Submission/EmployerInfo.cs]: Good catch. I updated them both.

File: DOL.WHD.Section14c.Domain/Models/Submission/WageTypeInfo.cs [hunk 4]
```diff
@@ -25,8 +27,23 @@ public WageTypeInfo()
...
+                        x =>
+                            new WageTypeInfoSCAAttachment
+                            {
+                                SCAAttachmentId = x,
+                                WageTypeInfoId = Id,
+                            }).ToList();
+                }
+            }
+        }
+        public virtual ICollection<WageTypeInfoSCAAttachment> SCAAttachment { get; set; }
```
[the remark this answers is not in the document]
Comment (dciceman) [reply DOL.WHD.Section14c.Domain/Models/Submission/WageTypeInfo.cs]: Fixed

**Pull request merged**